pkgs/development/compilers/open-watcom/v2.nix

```diff
@@ -89,6 +89,8 @@ stdenv.mkDerivation rec {
     prettyName = "open-watcom-v2";
     updateScript = unstableGitUpdater {
       url = "https://github.com/open-watcom/open-watcom-v2.git";
+      # no numerical releases, monthly "YYYY-MM-DD-Build" tags and daily "Current-build", "Last-CI-build" & "Coverity-scan" retagging
```

Comment:
Similar to `nix-search-cli` in terms of numeric tags, but the 3 latest ones are always numberless tags that get continuously recycled by CI. No real information one can extract from that system, so I think `0` is perfectly adequate here.